flask/app/models.py

```diff
@@ -419,14 +419,28 @@ class PipelineDatasets(db.Model):
 
 @dataclass
 class Gene(db.Model):
-    gene_id: int = db.Column(db.Integer, primary_key=True)
-    hgnc_gene_id: int = db.Column(db.Integer, unique=True)
-    ensembl_id: int = db.Column(db.Integer, unique=True)
-    gene: str = db.Column(db.String(50))
-    hgnc_gene_name: str = db.Column(db.String(50))
+    # Truncate ENSG
+    ensembl_id: int = db.Column(db.Integer, primary_key=True)
+    # Truncate HGNC:
+    hgnc_id: int = db.Column(db.Integer, unique=True)
+    ncbi_id: int = db.Column(db.Integer, unique=True)
     variants = db.relationship("Variant", backref="gene")
 
 
+@dataclass
+class GeneAlias(db.Model):
+    # Autoincrement surrogate key
+    alias_id = db.Column(db.Integer, primary_key=True)
+    ensembl_id: int = db.Column(
+        db.Integer,
+        db.ForeignKey("gene.ensembl_id", onupdate="cascade", ondelete="cascade"),
+    )
+    # Not unique in case one name corresponds to multiple ENSGs across releases
+    name: str = db.Column(db.String(20), nullable=False)
```

Comment:
Increase length to 30 or even 50?

https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:52873
https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:42360